test/integration/serverTest.js

```diff
@@ -129,6 +130,30 @@ async function im21Door(doorCoreId, signal) {
   }
 }
 
+async function normalHeartbeat(coreId) {
+  try {
+    await chai.request(server).post('/api/heartbeat').send({
+      coreid: coreId,
+      data: `{"doorMissedMsg": 0, "doorLowBatt": false, "doorLastHeartbeat": 1000, "resetReason": "NONE", "states":[]}`,
+    })
+    await helpers.sleep(50)
+  } catch (e) {
+    helpers.log(e)
+  }
+}
+
+async function lowBatteryHeartbeat(coreId) {
+  try {
+    await chai.request(server).post('/api/heartbeat').send({
```

Comment:
Do we have any tests for low battery being sent in via the `/api/door` endpoint?

Reply:
we do now